docker.go

```diff
@@ -276,6 +285,15 @@ func commandBuild(build Build) *exec.Cmd {
 	return exec.Command(dockerExe, args...)
 }
 
+func commandLoad(location string) *exec.Cmd {
```

Comment:
"loadDockerImageFromFile" name would be more suiting for this function, wouldn't it?

Reply:
The convention seams to be "command" + actual Docker command, so I'm following that. Actually the name you've suggested implies sideffects, whereas the function deterministically produces a struct - it's a command pattern implementation